### DISCUSSION TITLE
续费后变免费版，邮箱授权功能无效

### DISCUSSION THREAD

**Discussion by daluzhuzi** [getlantern/lantern-forum-cn#840] [General]
8月7日续费，14日左右变成免费版，向邮箱授权功能无效，请解决
注册邮箱1982722800@qq.com，之前发过贴邮箱记错为kknd.jr@163.com，请把激活码发到其中一个，谢谢。
续费凭证
![l1](https://github.com/user-attachments/assets/42fe7792-76cf-4d32-83cf-8014c436fa71)

**Comment by MaxMKMax** [comment]
@daluzhuzi 您好. 我已经帮您提交问题上去了. 周么美国团队是不上班的, 要等下周一晚上了. 他们是美国时间. 
然后我这里是没权限帮您解决问题的. 只能帮您提交问题和对接他们. 希望您理解.
不好意思让您久等了. 谢谢您的耐心等待. 谢谢您支持蓝灯!
